project/Build.scala

```diff
@@ -79,7 +83,9 @@ object DottyBuild extends Build {
 
       tuning ::: agentOptions ::: travis_build ::: fullpath
     }
-  )
+  ) ++ (if (isTravisBuild) Seq() else addCommandAlias("test", "partest"))
```

Comment:
I see. Does this need to be the `test` target or would it work with another target (say `nonpartest`)? I briefly looked at this already, but couldn't get it to work as I couldn't use test frameworks from my own key or set it up the same as Test and test... But I'll try again.

Alternatively, if you need to edit the Build.scala anyway to uncomment agentOptions, I could also add a flag to set agentOptions and use JUnit in that case. That would be much simpler, but wouldn't give a way to run JUnit without `reload`.